JiraPS/Public/Invoke-JiraMethod.ps1

```diff
@@ -3,7 +3,7 @@ function Invoke-JiraMethod {
     [CmdletBinding( SupportsPaging )]
     param(
         [Parameter( Mandatory )]
-        [Uri]
+        [psobject]
```

Comment:
`[Uri]` also supports relative paths. Why change the type of the variable?

Reply:
It is my background of .Net application development.

Uri class has a few [constructors](https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=netframework-4.8#System_Uri__ctor_System_String_), but any of them does not allow to create relative link without specify UriKind.
Therefore, the following examples will throw an exception: `[uri]::new("/test/a")` or ` New-Object uri "/test/a"`.

When I investigate all dependencies to Invoke-JiraMethod I found that often the module use string representation of URL, not an Uri instance.
So, in order to make our live easy and avoid the error I change the type to PSObject.

But, after I received you question I found that convert operation in PowerShell allow to convert relative path to Uri instance without specify UriKind: `[uri]"/test/a"`

In short: I will provide a new commit where I will change type back to URI.